docs_user/modules/proc_completing-prerequisites-for-migrating-ceph-monitoring-stack.adoc

```diff
@@ -2,19 +2,22 @@
 
 = Completing prerequisites for a {Ceph} cluster with monitoring stack components
 
-You must complete the following prerequisites before you migrate a {Ceph} cluster with monitoring stack components.
+Complete the following prerequisites before you migrate a {Ceph} cluster with monitoring stack components.
```

Comment:
Everything is commented out below this sentence. Should this be removed?

Reply:
I'm going to be working with the Ceph engineer to create a separate Ceph prerequisites module that will go in the first chapter of the adoption guide. I'm hiding this note to preserve it so I can eventually put it in the yet-to-be-created module.